src/sharedInterfaces/queryResult.ts

```diff
@@ -27,12 +66,37 @@ export interface QueryResultWebviewState {
     resultSetSummaries: { [key: number]: ResultSetSummary };
     messages: IMessage[];
     tabStates?: QueryResultTabStates;
+    isExecutionPlan?: boolean;
+    executionPlanState: {
+        sqlPlanContent?: string;
+        executionPlan?: GetExecutionPlanResult;
+        executionPlanGraphs?: ExecutionPlanGraph[];
+        theme?: string;
```

Comment:
Theming variable is provided by vscodeWebviewProvider. Please try to use that instead. Things like theming, telemetry and localization should be kept centralized as any bug fixes made to them will be applied everywhere.